tst/standard/examples.tst

```diff
@@ -226,6 +226,23 @@ Error, the arguments <n> and <k> must be non-negative integers,
 gap> JohnsonDigraph(IsMutableDigraph, 4, 2);
 <mutable digraph with 6 vertices, 24 edges>
 
+# BananaTreeGraph
+gap> D := BananaTreeGraph(2, 4);
+<immutable connected symmetric digraph with 9 vertices, 16 edges>
+gap> D := BananaTreeGraph(3, 3);
+<immutable connected symmetric digraph with 10 vertices, 18 edges>
+gap> D := BananaTreeGraph(5, 2);
+<immutable connected symmetric digraph with 11 vertices, 20 edges>
+gap> D := BananaTreeGraph(3, 4);
+<immutable connected symmetric digraph with 13 vertices, 24 edges>
+gap> D := BananaTreeGraph(0, 0);
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
+Error, no 1st choice method found for `BananaTreeGraph' on 2 arguments
+gap> D := DigraphMutableCopy(BananaTreeGraph(2, 4));
+<mutable digraph with 9 vertices, 16 edges>
```

Comment:
```suggestion
```
I think you should just delete this test, since it's not testing anything about your new code that the first test on 230 isn't testing.